### PR TITLE
Single page

### PR DESCRIPTION
**Pull request by pbb34**
ccontains to main files now
index.html
index.js

### CODE REVIEW

File: src/www/index.html [hunk 1]
```diff
@@ -38,7 +140,429 @@ <h4 align="center"> Icebreaker App</h4>
...
+<!--------------------------------------------Page 4 begin here------------------------------------------------------>
+
+<div data-role="page" id="select_topics" data-theme="a" align="center" class="ui-field-contain">
+    <div class="slideshow-container">
+        <div data-role="header">
+            <a href="/index.html" data-icon="delete">Cancel</a>
+            <h3>Select topics </h3>
+        </div>
+
+        <!-- ========================================Topic 1=================================================================-->
```
Comment (catvec) [inline src/www/index.html]: only have a topic entry for 1 topic. Don't manually code in an entry for each topic. Instead we will an HTTP request to the get all topics endpoints.

**Review by catvec — CHANGES_REQUESTED** [review]
1 small thing but otherwise good.  

Thanks for making a more descriptive branch name. Try to strike a balance between describing what the branch is and keeping it short.

File: src/www/index.html [hunk 2]
```diff
@@ -2,17 +2,114 @@
 <html lang="en" xmlns="https://www.w3.org/1999/html">
 <head>
     <meta charset="UTF-8">
+
+    <style>
+        * {box-sizing: border-box}
```
Comment (catvec) [inline src/www/index.html]: Put this in this `.css` files.

**Review by catvec — CHANGES_REQUESTED** [review]
1 more thing.

File: src/www/index.html (hunk 2 above)
Comment (pbb34) [reply src/www/index.html]: ok, it was supposed to be in there but latest code i pulled from master probably took it out

**Review by catvec — APPROVED** [review]
good to merge.

**Pull request merged**